packages/website/content/docs/overview/getting-started.md

```diff
@@ -85,4 +85,4 @@ In the directory where you run flamegrill, there should be an `AppTest.html` fil
 
 ![flamegraph](./results.png)
 
-We can see here that `InefficientComponent` stands out quite a bit, consuming 47.62% of total render time.
+We can see here that `InefficientComponent` stands out quite a bit, consuming 47.62% of total render time.t
```

Comment:
The changes in this file seem unintentional.

Reply:
Excellent catch, thanks!